### DISCUSSION TITLE
Can't Run .41

### DISCUSSION THREAD

**Discussion by dwt1978** [rejetto/hfs#175] [General]
Downloaded the latest version (running Windows 7) .41. When I run the file a window flashes into view for a second and then is gone. HFS is not running. Repeat and the same thing happens. Any idea what's going on?

**Comment by rejetto** [comment]
You may have an error. Run it from the command line, so you can read it

**Reply by dwt1978** [reply]
How do I do that? Been like 20 years since I used the command line for anything.  :)

**Comment by rejetto** [comment]
Ok, first check if there is a debug.log file in the folder

**Reply by dwt1978** [reply]
There is no debug.log file in the folder. Except for the very brief flash of some kind of window, there's no indication that the server was even turned on. I just tried it again a few times and I'm certain it's a command prompt window that flashes briefly.

**Comment by rejetto** [comment]
i'm trying to avoid to ask you to use command line :)

1. download and unzip this file [run-hfs.bat.zip](https://github.com/rejetto/hfs/files/11069842/run-hfs.bat.zip)
2. you get a file named "run-hfs.bat" out of it (no .zip at the end): put it in the hfs folder
3. double click it

now you should be able to see what's written in the window before it closes

**Reply by dwt1978** [reply]
Well, I am really disappointed. The HFS does not run on Windows 7 any more. And, the message says any issues won't be fixed. A sad day for me. Here's the message.
"Node.js is only supported on Windows 8.1, Windows Server 2012 R2, or higher.
Setting the NODE_SKIP_PLATFORM_CHECK environment variable to 1 skips this
check, but Node.js might not execute correctly. Any issues encountered on
unsupported platforms will not be fixed.Press any key to continue . . ."
I'd like to try setting it to skip the OS check as mentioned. Anyone know the steps to do this?

**Comment by rejetto** [comment]
didn't see that coming :(
I'm sorry, but this version of HFS runs thanks to this other software, called "node.js".
Maybe someone can work on this problem and find a solution.
It is too much effort for me, at the moment. I can barely handle the rest.

**Reply by dwt1978** [reply]
That's okay. Part of the price I pay for sticking with Windows 7. An older version of HFS will work for me. There's a few tweaks I need to make to the web template and I need to figure out how to stop access except to those with passwords. I was lookijng around for a tutorial on the subject, but haven't found one yet.

**Comment by rejetto** [comment]
also in hfs 2, you first create an account and then set permissions on the folder you want (even "home")
